modules/API/external/external_api.js

```diff
@@ -366,6 +366,23 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
         return iframe.contentWindow.document.getElementById('largeVideo');
     }
 
+    /**
+     * Getter for participant specific video element in Jitsi Meet
+     * 
+     * @param {string} participantId - Participant id
+     * or "local" for local video
```

Comment:
I get eslint error for missing dot in the end of the sentence.

Reply:
Fixed.